### ISSUE TITLE
VET failing

### ISSUE THREAD

**Issue by lauranuttall** [gwpy/vet#19]
I tried running VET and it's failing due to the following:

```pytb
Loading configuration...
Traceback (most recent call last):
  File "/home/detchar/opt/gwpysoft-2.7/bin/gw_summary", line 470, in <module>
    plugins = config.load_plugins()
  File "/home/detchar/opt/gwpysoft-2.7/lib/python2.7/site-packages/gwsumm/config.py", line 235, in load_plugins
    mods.append(import_module(plugin))
  File "/usr/lib64/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/detchar/opt/gwpysoft-2.7/lib/python2.7/site-packages/gwvet/__init__.py", line 31, in <module>
    from .metric import *
  File "/home/detchar/opt/gwpysoft-2.7/lib/python2.7/site-packages/gwvet/metric/__init__.py", line 328, in <module>
    from metrics import *
  File "/home/detchar/opt/gwpysoft-2.7/lib/python2.7/site-packages/gwvet/metric/metrics.py", line 34, in <module>
    from gwpy.table.utils import get_table_column
ImportError: No module named utils
```

**Comment by duncanmmacleod** [comment]
@lauranuttall, the `gwpy.table.utils` module was removed in gwpy-0.12.0 (or earlier), any calls should be replaced with direct column access (e.g. `table[column]`).

**Comment by duncanmmacleod** [comment]
In fact, this error was fixed already in ecaf13654ecb73518547bb4d3703a28159c0f708, the `gwpysoft-2.7` installation just needs to be updated.
